### PR TITLE
Add crud endpoints

### PR DESCRIPTION
**Pull request by timothydadams**
@JasperC1999 @jayrbarr

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -10,28 +10,75 @@ app.use(express.static(`${__dirname}/../client/public`));
...
-// ROUTES
-app.get('/api/reviews/:search', (req, res) => {
-  db.find(req.params.search, (err, results) => {
+// OLD FRONT END ROUTES
+app.patch('/api/reviews/:id', ({ params: { id }, body: { reviewId, field, value } }, res) => {
+  db.update(id, reviewId, field, value, (err, results) => {
     if (err) {
-      res.status(400).send(err);
+      res.sendStatus(400);
     } else {
```
Comment (jayrbarr) [inline server/index.js:18]: I remember Robin saying, "don't ever send errors from the back end to the client" or something like that. Errors can contain too much information. So good job changing that!

File: server/index.js [hunk 2]
```diff
@@ -10,28 +10,75 @@ app.use(express.static(`${__dirname}/../client/public`));
...
+  db.findId(id, (err, results) => {
+    if (err) {
+      console.log(err);
+      res.sendStatus(400);
+      return;
+    }
+    res.send(results);
+  });
+});
+
```
Comment (jayrbarr) [inline server/index.js:60]: How is this differentiated from the other GET route? Won't there be a conflict?

**Review by jayrbarr — APPROVED** [review]
Approving to keep you moving, but can you help me understand the two GET routes?

File: server/index.js (hunk 1 above)
Comment (timothydadams) [reply server/index.js:18]: thanks @jayrbarr !

**Review by jayrbarr — APPROVED** [review]
I like it. lgtm

File: server/index.js (hunk 2 above)
Comment (timothydadams) [reply server/index.js:60]: all new CRUD paths are preceded with /new-api/ now to maintain front-end functionality.

**Pull request merged**